libraries.gradle

```diff
@@ -100,6 +100,8 @@ ext {
             oracle:          'com.oracle.jdbc:ojdbc7:12.1.0.2',
             mssql:           'com.microsoft.sqlserver:mssql-jdbc:6.1.0.jre8',
             db2:             'com.ibm.db2:db2jcc:10.5',
+            //hana:            'com.sap.db.jdbc:ngdbc:1.120.20',
+            hana:            'com.sap.db.jdbc:ngdbc:2.2.1',
```

Comment:
Is this jar not publicly available?

Reply:
You mean the version 1.120.20? I added it so people using HANA 1 know what the minimum supported version of the client is. If you prefer I can just add a comment stating this instead of the commented out line or leave it out completely.